USAGE.md

```diff
@@ -210,10 +210,6 @@ If so, you will want to stop the bleeding and prevent more violations from occur
 
     bin/packwerk update-todo
 
-Similar to `bin/packwerk check`, you may also run `bin/packwerk update-todo` on folders or packages:
-
-    bin/packwerk update-todo components/your_package
-
 ![](static/packwerk_update.gif)
 
 _Note: Changing dependencies or enabling dependencies will not require a full update of the codebase, only the package that changed. On the other hand, changing or enabling privacy will require a full update of the codebase._
```

Comment:
Is this note related to the removed sentence above?
"full update" is the only option now, isn't it?

```suggestion
```

Reply:
True, we can remove this too 👍